### ISSUE TITLE
get_agasc_cone segmentation fault

### ISSUE THREAD

**Issue by malgosias** [sot/agasc#19]
>from agasc import get_agasc_cone
>ra = 359.25177
>dec = 0.1305667
>get_agasc_cone(ra, dec, 0.1)

<works OK>

but

>ra = np.float16(359.25177)
>dec = np.float16(0.1305667)
>ra, dec

(359.25, 0.13062)

>get_agasc_cone(ra, dec, 0.1)
Segmentation fault (core dumped)

**Comment by taldcroft** [comment]
Probably in the `numexpr` code.   Maybe something like `np.radians(ra1).astype(np.float64)` would do the trick, but haven't tested that.

**Comment by jeanconn** [comment]
That does seem to be where the segfault happens.  Not sure if an astype or cast makes more sense or if this needs np.float64 or np.float32
```
In [13]: %timeit np.radians(np.float16(10)).astype(np.float64)
4.48 µs ± 10 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [14]: %timeit np.radians(np.float64(np.float16(10)))
2.26 µs ± 5.52 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [21]: %timeit np.float64(np.radians(np.float16(10)))
2.21 µs ± 4.4 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

```

**Comment by jeanconn** [comment]
I think this was closed by #20
